### PR TITLE
Update German strings.

### PR DESCRIPTION
**Pull request by hybridOL**
I've updated translations for German. Some typos fixed and some wording straightened.

### CODE REVIEW

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

File: tools/unix/build_omim.sh [hunk 1]
```diff
@@ -66,8 +66,8 @@ if [ "$(uname -s)" == "Darwin" ]; then
   SPEC=macx-clang
   PROCESSES=4
 else
-  SPEC=linux-clang
-  PROCESSES=$(($(nproc) / 2))
+  SPEC=linux-clang-libc++
```
Comment (Zverik) [inline tools/unix/build_omim.sh]: Could you please extract this change to a separate pull request?

**Comment by Zverik** [comment]
Please sign [My.com CLA](http://maps.me/cla/), so we can merge your changes.

File: tools/unix/build_omim.sh (hunk 1 above)
Comment (hybridOL) [inline tools/unix/build_omim.sh]: If only I knew how to separate commits...

**Comment by hybridOL** [comment]
CLA signed

File: tools/unix/build_omim.sh (hunk 1 above)
Comment (Zverik) [inline tools/unix/build_omim.sh]: As I can see, you change `build_omim.sh` in two separate commits. These need to be moved to another branch, from which you would make a separate pull request.
* First, do `git checkout upstream/master; git checkout -b fix_build`
* Then you need to cherry-pick your commits: copy their SHA codes and do `git cherry-pick 12312123`, where 12312123 is a SHA code.
Your branch is then ready, you can do `git push` and submit a new pull request.

As for the old one, you need to remove two commits. First, check out this branch. Then, you can either:
* Restore `build_omim.sh` to its previous state (`git reset HEAD~3 build_omim.sh`, or just copy-paste old lines) and commit this change;
* Or do a `git rebase -i HEAD~3` and delete two commits in which you change this file. After that you would need to do `git push --force`, which is not a safe option, but works good most of the time.

Comment (hybridOL) [inline tools/unix/build_omim.sh]: Ok, thanks. I wasn't sure whether push --force would work in this setting. This one's fixed now and clean. Branch pull request will follow
